metaflow/plugins/conda_escape/__init__.py

```diff
@@ -0,0 +1,162 @@
+# Portions of this plugin are inspired by RPyC
```

Comment:
I don't think this plugin has anything specific to `conda`. Maybe, we can just use `escape_hatch` as the directory name, rather than `conda_escape`.